osbs/http.py

```diff
@@ -81,13 +97,22 @@ class HttpStream(object):
     def __init__(self, url, method, data=None, kerberos_auth=False,
                  allow_redirects=True, verify_ssl=True, ca=None, use_json=False,
                  headers=None, stream=False, username=None, password=None,
-                 client_cert=None, client_key=None, verbose=False):
+                 client_cert=None, client_key=None, verbose=False, retries_enabled=True):
         self.finished = False  # have we read all data?
         self.closed = False    # have we destroyed curl resources?
 
         self.status_code = 0
         self.headers = None
 
+        retry = OSBSRetry(
```

Comment:
We can pass a custom list of statuses in using `Retry(..., status_forcelist=...)` can't we?